docs/contributing/testing.md

```diff
@@ -1,23 +1,20 @@
 ---
 id: testing
-title: Contributing
+title: Testing
 sidebar_label: Testing
 ---
 
-# Testing
-
-When building Web sites and apps on FAST-DNA it's recommended to perform exhaustive testing to instill the confidence that you've met your customers' expectations. The checklist below was compiled over the last few years and is a good starting point for manual testing Web applications.
-
 ## Checklist
+This testing checklist is a good starting point for manual testing Web applications.
```

Comment:
I'm sure this comes up again, but I vote in lowercasing all references to "web". So "websites", "web applications", "web page", etc... 
Feels cleaner to me, and, in general, this is how I see it written in the various communities.

Reply:
Changed.